server.go

```diff
@@ -236,11 +184,7 @@ func checkDefaultSecurityTests(configAPI *apiContext.APIConfig) error {
 	bandit, err := analysis.FindOneDBSecurityTest(banditQuery)
 	if err == mgo.ErrNotFound {
 		// As Bandit securityTest is not set into MongoDB, HuskyCI will insert it.
-		if errLog := glbgelf.Logger.SendLog(map[string]interface{}{
-			"action": "checkDefaultSecurityTests",
-			"info":   "SERVER"}, "ERROR", "Bandit securityTest not found!"); errLog != nil {
-			fmt.Println("glbgelf error: ", errLog)
-		}
+		log.Warning("checkDefaultSecurityTests", "SERVER", 204)
```

Comment:
This warning log message looks right. This is a nice example of the purpose of a warning message.

Reply:
😃 Nice!